tests/unit_tests/detection_metrics_distance_based_test.py

```diff
@@ -284,7 +271,7 @@ def test_random_case_generation_and_verification(self):
         expected_mAP, expected_f1 = self.calculate_expected_metrics(self, target_precision, target_recall)
 
         # Validate the results
-        self.validate_results(results, precision=target_precision, recall=target_recall, mAP=expected_mAP, F1=expected_f1)
+        self.validate_results(results, precision=target_precision, recall=target_recall, mAP=None, F1=expected_f1)
```

Comment:
Why set `mAP=None` ?

Reply:
In the previous version the mAP was not checked at all for all cases: https://github.com/Deci-AI/super-gradients/pull/1696/files#diff-9739186c39900e4bbc707a4059031225ac260667e5814fc2fdfb5a896e3a7502L90

In the remaining place the random generation of targets is somewhat non-deterministic, and mAP and since the confidence of predicted boxes is always 1 then their order actually impacts the mAP score. So for this specific case mAP is not checked and in all other cases where we have deterministic boxes it is checked